libs/qlik/src/dashboard-page/vl-qlik-dashboard-page.component.ts

```diff
@@ -0,0 +1,285 @@
+import {CSSResult, html, PropertyDeclarations} from 'lit';
+import {BaseLitElement, registerWebComponents, webComponent} from '@domg-wc/common-utilities';
+import {vlElementsStyle} from "@domg-wc/elements";
+import '../dashboard';
+import {DashboardVisualization, Filter, Views} from "../utils/vl-qlik.types";
+import {exportCSVFile, exportExcelFile, Qlik} from "@domg/qlik-lib";
+import {
+    bindVlSelect,
+    comparingWithFunction,
+    performWithLoader,
+    queryById,
+    renderStack
+} from "../utils/qlik-render-helpers";
+import "../dashboard/vl-qlik-dashboard.component";
+
+import {VlAlert, VlAnnotation, VlLoaderComponent} from "@domg-wc/components";
+
+@webComponent('vl-qlik-dashboard-page')
+export class VlQlikDashboardPageComponent extends BaseLitElement {
+    public title = '';
```

Comment:
de conventie is dat 'public' niet geschreven wordt omdat dat de default is in typescript
we proberen wel alles wat kan 'private' (of 'protected') te zetten - dus die moet je specifieren